neuralprophet/plot_model_parameters_plotly.py

```diff
@@ -0,0 +1,669 @@
+    Returns:
+        A dictionary with Plotly traces, xaxis and yaxis
+    """
+    traces = []
+    zeroline_color = "#AAA"
+    color = "#0072B2"
+
+    n_lags = weights.shape[1]
+    lags_range = list(range(1, 1 + n_lags))[::-1]
+    if focus is None:
+        weights = np.sum(np.abs(weights), axis=0)
+        weights = weights / np.sum(weights)
+
+        traces.append(
+            go.Bar(
+                name=comp_name,
+                x=lags_range,
+                y=weights,
+                marker_color=color,
+            )
+        )
+
+    else:
+        if len(weights.shape) == 2:
+            weights = weights[focus - 1, :]
+
+        traces.append(go.Bar(name=comp_name, x=lags_range, y=weights, marker_color=color, width=0.8))
+
+    xaxis = go.layout.XAxis(title=f"{comp_name} lag number")
+
+    if focus is None:
+        ax.set_ylabel("{} relevance".format(comp_name))
+        yaxis = go.layout.YAxis(
+            rangemode="normal",
+            title=go.layout.yaxis.Title(text=f"{comp_name} relevance"),
+            zerolinecolor=zeroline_color,
+            tickformat=",.0%",
+        )
+        # ax = set_y_as_percent(ax)
+    else:
+        yaxis = go.layout.YAxis(
+            rangemode="normal",
+            title=go.layout.yaxis.Title(text=f"{comp_name} weight ({focus})-ahead"),
+            zerolinecolor=zeroline_color,
+        )
+
+    return {"traces": traces, "xaxis": xaxis, "yaxis": yaxis}
+
+
+def plot_yearly(m, comp_name="yearly", yearly_start=0, quick=True, multiplicative=False):
+    """Plot the yearly component of the forecast.
+
+    Args:
+        m (NeuralProphet): fitted model.
+        comp_name (str): Name of seasonality component if previously changed from default 'weekly'.
+        yearly_start (int): specifying the start day of the yearly seasonality plot.
+            0 (default) starts the year on Jan 1.
+            1 shifts by 1 day to Jan 2, and so on.
+        quick (bool): use quick low-evel call of model. might break in future.
+        multiplicative (bool): set y axis as percentage
+    Returns:
+        A dictionary with Plotly traces, xaxis and yaxis
+    """
+    traces = []
+    color = "#0072B2"
+    line_width = 1
+    zeroline_color = "#AAA"
+
+    # Compute yearly seasonality for a Jan 1 - Dec 31 sequence of dates.
+    days = pd.date_range(start="2017-01-01", periods=365) + pd.Timedelta(days=yearly_start)
+    df_y = pd.DataFrame({"ds": days})
+    if quick:
+        predicted = predict_season_from_dates(m, dates=df_y["ds"], name=comp_name)
+    else:
+        predicted = m.predict_seasonal_components(df_y)[comp_name]
+
+    traces.append(
+        go.Scatter(
+            name=comp_name,
+            x=range(
+                df_y["ds"].dt.to_pydatetime(),
+                y=predicted,
+                mode="lines",
+                line=dict(color=color, width=line_width),
+                fill="none",
+            ),
```

Comment:
```suggestion
            x=df_y["ds"].dt.to_pydatetime(),
            y=predicted,
            mode="lines",
            line=dict(color=color, width=line_width),
            fill="none",
```
Remove range(), datetime cannot be converted to a scalar index

Reply:
Fixed in a later commit